src/fstab-generator/fstab-generator.c

```diff
@@ -47,6 +47,12 @@
 #include "virt.h"
 #include "volatile-util.h"
 
+typedef enum MountpointFlags {
+        NOAUTO    = 1 << 0,
+        NOFAIL    = 1 << 1,
+        AUTOMOUNT = 1 << 2,
+} MountpointFlags;
```

Comment:
not that it matters, but maybe MountpointFlags → MountPointFlags?

Reply:
We use "mountpoint" as a word all over the codebase, so I think "MountpointFlags" is OK.